list_of_subprojects.md

```diff
@@ -0,0 +1,66 @@
+# List of Official Jupyter Subprojects
+
+At the highest level, any Github repository under any official Jupyter GitHub organization is either itself an Official Jupyter Subproject, or is part of an Official Jupyter Subproject. All such Subprojects have all of the privileges and responsibilities detailed in the Software Subprojects of Jupyter governance document. Within the Official Jupyter Subproject designation, there are two types of Subprojects: ones with a formal decision making body and SSC representation, and smaller, less active ones whose decision making body is the SSC itself. This document enumerates Subprojects of these two types.
+
+### Official Subprojects with SSC representation
+
+The following Jupyter Subprojects have their own formal decision making body that elects and maintains an SSC representative:
+
+- JupyterLab (https://github.com/jupyterlab/jupyterlab)
+- JupyterHub (https://github.com/jupyterhub/jupyterhub)
+- Voilà (https://github.com/voila-dashboards/voila)
+- Jupyter Server (https://github.com/jupyter-server/jupyter_server)
+- Jupyter Widgets (https://github.com/jupyter-widgets)
+- Jupyter Notebook (https://github.com/jupyter/notebook)
+- Binder
+  - https://github.com/jupyterhub/binder
+  - https://github.com/jupyterhub/binderhub
+- Jupyter Kernels
+  - jupyter-xeus (https://github.com/jupyter-xeus/)
+  - IPython (https://github.com/ipython)
```

Comment:
Personally I see IPython and IPykernel differently. I rarely contribute to IPykernel and trust Sylvain, Martin ... etc with it, though I'm mostly involve in IPython. Not sure if other agree.
```suggestion
  - IPython (https://github.com/ipython/ipython)
  - IPykernel (https://github.com/ipython/ipykernel)
```

Reply:
Since IPython isn't a kernel but does seem to naturally fit in the `Jupyter Kernels` group, designating `ipykernel` and `ipython` individually seems fine. I wouldn't want to enumerate every kernel under `jupyter-xeus`, though.

Reply:
That's fine with me, we can also move IPython into "Core" if you wish; as it is also relatively independent from Jupyter itself. It's just a suggestion as I expect the IPython and the IPykernel team to potentially be quite different.